bot/exts/summer_aoc.py

```diff
@@ -44,8 +44,6 @@
 
 NEXT_PUZZLE_TEXT = """
 The next puzzle will be posted <t:{timestamp}:R>.
-To receive notifications when new puzzles are released, run `!subscribe` in <#{bot_commands}> and assign yourself \
-the Revival of Code role.
```

Comment:
Perhaps we could instead explain what the new way is of subscribing to notifications?

Reply:
Sure, sounds good to me. Ill make the changes ASAP

Reply:
Should be added in the new [commit ](https://github.com/python-discord/sir-robin/pull/100/commits/8ec55098e7bd38eb4495f79726f86b8a2b7a9117)